### ISSUE TITLE
[1.19.2] Crash due to Incompat with Pinlib and Porting Lib

### ISSUE THREAD

**Issue by DoctorFrank28** [TeamTwilight/twilightforest-fabric#125]
### Fabric Loader Version

0.14.9

### Fabric API Version

0.62.0

### Twilight Forest Version

4.2.299

### Client Log

https://gist.github.com/DoctorFrank28/d93bcea56b8d263fe937f5ab8e7cab18

### Crash Report (if applicable)

no crash report

### Steps to Reproduce

i added the twilight

### What You Expected

.

### What Happened Instead

it crashes

### Please Read and Confirm The Following

- [X] I have confirmed this bug can be replicated without the use of Optifine.
- [X] I have confirmed the details provided in this report are concise as possible and does not contained vague information (ie. Versions are properly recorded, answers to questions are clear).
- [X] I have confirmed this does not happen on the forge version (If it does please report to **[here](https://github.com/TeamTwilight/twilightforest/issues)** instead).
- [X] I have confirmed this issue is unique and has not been reported already.

**Comment by FluffyBumblebees** [comment]
Why did u make another issue? U can comment on closed ones

**Comment by DoctorFrank28** [comment]
because its a different pack,

**Comment by Dragon-Seeker** [comment]
Seems to be a conflict between porting lib and PinLib:
```
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: 
@At("LOAD" ordinal=0 index=-1) on net/minecraft/class_330$class_331::pinlib$CheckForCustomMarker with
priority 1000 cannot inject into net/minecraft/class_330$class_331::method_1777(Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;ZI)V 
merged by io.github.fabricators_of_create.porting_lib.mixin.client.MapRendererMapInstanceMixin with
priority 1000 [PREINJECT Applicator Phase -> pinlib.mixins.json:MapTextureMixin from mod pinlib -> Prepare Injections ->  -> localvar$ibj000$pinlib$CheckForCustomMarker(Lnet/minecraft/class_20;)Lnet/minecraft/class_20; -> Prepare]
```

I'm guessing that the PortingLib Overwrite is not allowing for a valid injection spot

**Comment by Dragon-Seeker** [comment]
Just waiting on this [PR](https://github.com/Fabricators-of-Create/Porting-Lib/pull/16) to be merged to fix the incompat
